rfc6962-bis.xml

```diff
@@ -1226,10 +1226,10 @@ entry specified by <spanx style="verb">start</spanx>.
           </t>
         </section>
       </section>
-      <section title="TLS Feature Extension">
-	<t>
-	  A CA MAY include the <xref target="tls_transinfo_extension">transparency_info</xref> extension number in a <xref target="RFC7633">TLS Feature Extension</xref> in root or intermediate certificates. This indicates that CT compliance is required for all certificates which are directly or indirectly issued by such certificates.
-	</t>
+      <section title="TLS Feature Extension" anchor="tls_feature_extension">
+        <t>
+          A certification authority MAY include the <xref target="tls_transinfo_extension">transparency_info</xref> TLS extension identifier in the <xref target="RFC7633">TLS Feature</xref> certificate extension in root, intermediate and end-entity certificates. When a certificate chain includes such a certificate, this indicates that CT compliance is required.
```

Comment:
BTW, I changed it to "TLS extension identifier" because that's the language used by RFC7633.  However, I wonder if "TLS ExtensionType value" would be more accurate (see http://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml).